ci/deps/actions-38-downstream_compat.yaml

```diff
@@ -34,7 +34,7 @@ dependencies:
   - gcsfs
   - jinja2
   - lxml
-  - matplotlib
+  - matplotlib>=3.6.1
```

Comment:
I think we should do this for every dependency. cc @mroeschke

Reply:
Yeah sure.

Reply:
Agreed, and to update our min version script to account for these dependency files